compiler/quilt/tools/util.py

```diff
@@ -166,19 +169,21 @@ def is_nodename(string):
     :returns: True if string could be used as a node name, False otherwise
     :rtype: bool
     """
+    # TODO: Permit keywords once node['item'] notation is implemented
+    # TODO: Update the following description once node['item'] notation is implemented
     ## Currently a node name has the following characteristics:
     # * Must be a python identifier
-    # * May be a python keyword
+    # * Must not be a python keyword  (limitation of current implementation)
     # * Must not start with an underscore
     if string.startswith('_'):
         return False
-    return bool(PYTHON_IDENTIFIER_RE.match(string))
+    return is_identifier(string, permit_keyword=False)
 
 
-def to_identifier(string):
+def to_identifier(string, permit_keyword=False, strip_underscores=False):
```

Comment:
Are we ever planning to using both `permit_keyword=False` and `permit_keyword=True`?

If I understand correctly, it's always `False` right now, and you're planning to switch it to always `True` later? If so, it shouldn't be a parameter - just hardcode the implementation, and then later change it.

Reply:
Hrm.. ..perhaps not.  I'll hardcode it.